src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -314,6 +320,24 @@ public override AbstractFormattingRule WithOptions(AnalyzerConfigOptions options
                 }
             }
 
+            // For spacing after the 'not' pattern operator
+            if (previousToken.Parent.IsKind(SyntaxKindEx.NotPattern))
```

Comment:
hrmm... i wouldn't control this with SpacingAroundBinaryOperator.  For one, it's not a binary operator.  For another, it's just a word, and should likely just always be forced to single space (like we would with much of our other prefix-keyword constructs).

Reply:
Updated